lib/src/main/java/uk/gov/di/ipv/core/library/persistence/DataStore.java

```diff
@@ -7,7 +7,6 @@
 import software.amazon.awssdk.enhanced.dynamodb.model.QueryConditional;
 import software.amazon.awssdk.regions.Region;
 import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
-import uk.gov.di.ipv.core.library.service.ConfigurationService;
```

Comment:
I like that this is no longer given the entire ConfigurationService when it only needs a single value 👍